.rubocop.yml

```diff
@@ -7,3 +7,6 @@ AllCops:
 
 ParameterLists:
   CountKeywordArgs: false
+
+Metrics/ModuleLength:
```

Comment:
since you've updated the cocoa pods config, please back out this part of the change